src/write.c

```diff
@@ -1623,9 +1623,9 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             avifBoxMarker elst = avifRWStreamWriteFullBox(&s, "elst", AVIF_BOX_SIZE_TBD, 1, elstFlags);
             avifRWStreamWriteU32(&s, 1);                          // unsigned int(32) entry_count;
             avifRWStreamWriteU64(&s, framesDurationInTimescales); // unsigned int(64) segment_duration;
-            avifRWStreamWriteU64(&s, 0);                          // unsigned int(64) media_time;
-            avifRWStreamWriteU16(&s, 1);                          // unsigned int(16) media_rate_integer;
-            avifRWStreamWriteU16(&s, 0);                          // unsigned int(16) media_rate_fraction = 0;
+            avifRWStreamWriteU64(&s, 0);                          // int(64) media_time;
```

Comment:
I take it the fields are ints and the comment may have from the lack of signed write calls?

Reply:
I don't know where Vignesh copied the comments from. I checked the following three standards:

- ISO/IEC 23008-12:2017 (HEIF)
- ISO/IEC 14496-12:2020 (ISO BMFF)
- ISO/IEC 14496-12:2022 (ISO BMFF)

In these three standards, these fields are ints. They only differ in whether `media_rate_fraction` has the `= 0`.

Reply:
yes, sorry these are "int" fields. 

i mistakenly copy pasted them from the above lines as unsigned int.